test/deprecated/arrayFind.chpl

```diff
@@ -0,0 +1,6 @@
+// Marked unstable while we figure out if we want to change the interface to
```

Comment:
Is this comment right? Is it that the new interface for `find` still unsettled or is that we've settled on a new interface but are using the `useNewArrayFind` `config param` until we can get rid of the old interface after going through the normal deprecation cycle.

Reply:
I never read comments.  :D  I've updated this from the original copy-pasted comment.

Reply:
> I never read comments. :D I've updated this from the original copy-pasted comment.

Which is why clear code is always better than good comments: comments get outdated, don't get locked down with tests, and are a violation of the whole "don't repeat yourself" principle of software engineering; though I can't bring myself to say they're worthless /rant.